src/login.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { getAuth, signInWithEmailAndPassword, signOut } from "firebase/auth";
+import Typography from "@mui/material/Typography";
+import TextField from "@mui/material/TextField";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import './index.css';
+
+export default function LoginPage() {
+  const [password, setPassword] = useState<string>("");
+  const [username, setUsername] = useState<string>("");
+  const [signedIn, setSignedIn] = useState<boolean>(false);
+  const auth = getAuth();
+
+  const attemptLogin = async () => {
+
+    signInWithEmailAndPassword(auth, username, password)
+      .then((userCredential: any) => {
+      // Signed in
+      alert("success!");
+      setUsername("");
+      setPassword("");
+      setSignedIn(true);
+
+    })
+    .catch((error: any) => {
+      console.log(error.code);
+      console.log(error.message);
+    });
```

Comment:
If we go into this catch, can we alert to user with a simple error message? For example if it's a `auth/invalid-credential` can we tell the user that their username/password is invalid?

I attempted to login and can't tell if it's working or not and had to check the debug console to realize I was using the wrong password.